client/posts/posting/image.ts

```diff
@@ -0,0 +1,32 @@
+// Reload image if error
+// Handle event if image error
+async function onImageErr(e: Event) {
+    const el = (e.target as HTMLImageElement)
+
+    if (el.tagName !== "IMG" || (el.complete && el.naturalWidth !== 0)) {
+        return
+    }
+
```

Comment:
I meant here.

Reply:
If you're talking about that, we should be able to remove passive, and keep preventDefault, and have the same functionality.